bin/og_analytics.py

```diff
@@ -267,6 +288,8 @@ def getStats(self, start_date, end_date, og_type):
             for [url, count] in data:
                 id = url.split('/')[-1]
                 id = id.split('&')[0]
+		# added by Rabia 2020-01-29 - removes querystring parameters
+		id = id.split('?')[0]
```

Comment:
tabs instead of spaces?

Reply:
Yes, I didn't notice on my local dev since it happens to line up and script ran. Rest of script uses spaces, so I am going to replace tabs with spaces